.github/workflows/nightly.yml

```diff
@@ -30,7 +30,9 @@ jobs:
         with:
           script: |
             const gitHubLib = await import("${{ github.workspace }}/.github/workflows/github.mjs");
-            return await gitHubLib.dispatchRetryWorkflow(core, github.rest.actions, context, "${{ github.head_ref }}", 5);
+
```

Comment:
```suggestion
```

Reply:
If you are fine with it I would leave the empty line in between as it separates imports and code (as you would usually expect for js files as well.